build.sbt

```diff
@@ -89,82 +36,25 @@ lazy val cross = crossProject(JSPlatform, JVMPlatform)
         case _                         => Nil
       }
     },
-    Test / publishArtifact := false,
-    pomIncludeRepository := { _ => false }
-  )
-  .jvmSettings(
-    Compile / unmanagedSourceDirectories ++= scalaVersionSpecificJVMFolder(
-      "main",
-      (Compile / baseDirectory).value.getParentFile(),
-      scalaVersion.value
-    ),
-    Test / unmanagedSourceDirectories ++= scalaVersionSpecificJVMFolder(
-      "test",
-      (Test / baseDirectory).value.getParentFile(),
-      scalaVersion.value
-    )
+    mimaPreviousArtifacts ~= { _.filterNot(_.revision == "1.0.1") }
   )
   .jsSettings(
-    crossScalaVersions := (ThisBuild / crossScalaVersions).value.filter(_.startsWith("2")),
-    publishConfiguration := publishConfiguration.value.withOverwrite(true),
-    mimaBinaryIssueFilters ++= {
```

Comment:
Hmm, I don't quite understand changes with `MiMa`. Are we still checking it as previously? Just because `mimaBinaryIssueFilters` is gone, I feel some doubts.

Reply:
No, we are not checking it as previously :) Your mima was misconfigured, so that you were comparing your current JS code to the previous JVM artifacts. That's why you needed to filter all the JVM methods. Now, it is correctly configured, so that JS is compared to old JS artifacts.

Reply:
Uh oh, it seems you are right :( Thanks for the fix.

Reply:
No worries, it is too easy to mess this up. This is why sbt-typelevel is so important and I'm trying to move all the builds.

Most projects weren't even checking MiMa on JS, so good on you here ;)